src/controller/org.controller/org.controller.js

```diff
@@ -573,12 +583,19 @@ async function updateUser (req, res, next) {
     }
     if ('active_roles.remove' in req.ctx.query) {
       if (Array.isArray(req.ctx.query['active_roles.remove'])) {
-        req.query['active_roles.remove'].forEach(r => {
+        req.ctx.query['active_roles.remove'].forEach(r => {
           removeRoles.push(r)
         })
       }
     }
 
+    // updating user's roles and org_uuid is only allowed for secretariats and org admins
+    if ((active || removeRoles.length || addRoles.length || newUser.org_UUID) && (!isSecretariat && !isAdmin)) {
```

Comment:
Same comment as the previous one. This check does not make sure the Admin user belongs to the same org as the user being updated.